scripts/csi_test/csi_secrets

```diff
@@ -68,4 +68,4 @@ NodeUnstageVolumeSecret:
  management_address: STORAGE_ARRAYS
  username: USERNAME
  password: PASSWORD
- alias_in_storageclass: arr1
+ alias_in_storageclass: arr1
```

Comment:
![image](https://user-images.githubusercontent.com/57443811/104892379-c6230c80-597a-11eb-8587-a5c1c3a58c11.png)

looks like we [need a ticket](https://github.com/IBM/ibm-block-csi-driver/pull/250#discussion_r516813905) to enforce this

Reply:
done in https://github.com/IBM/ibm-block-csi-driver/pull/281 - resolving this conversation